src/Mono.Android/Java.Interop/TypeManager.cs

```diff
@@ -212,6 +212,10 @@ static Exception CreateJavaLocationException ()
 		[MethodImplAttribute(MethodImplOptions.InternalCall)]
 		static extern Type monodroid_typemap_java_to_managed (string java_type_name);
 
+		private static bool TypeRegistrationFallbackIsEnabled { get; } = InitializeTypeRegistrationFallbackIsEnabled ();
+		private static bool InitializeTypeRegistrationFallbackIsEnabled () =>
```

Comment:
Can you drop the `private` keyword please? :) This is the default visibility, there's no need to specify it explicitely

Reply:
Updated.